x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/InferenceBaseRestTest.java

```diff
@@ -511,7 +492,7 @@ protected void assertNonEmptyInferenceResults(Map<String, Object> resultMap, int
         }
     }
 
-    protected static void assertOkOrCreated(Response response) throws IOException {
+    static void assertOkOrCreated(Response response) throws IOException {
```

Comment:
```suggestion
    static void assertStatusOkOrCreated(Response response) throws IOException {
```

Wasn't 100% sure at a first glance what `Ok` or `Created` meant in this context. Maybe adding `Status` or even `HttpStatus` would make this more explicit?